### ISSUE TITLE
Lab 7 Exercise 1 Task 4 - list of flights is not shown

### ISSUE THREAD

**Issue by NFGodoy** [MicrosoftLearning/20487-DevelopingWindowsAzureAndWebServices#92]
I am following the LAK, as there are some discrepancies with the higher level instructions.
Lab7 - Exercise 1 - Task 4 - Step 40
No error messages, but nothing is shown. I tried Seattle, Paris, Rome.
I checked twice step-by-step the instructions. The booking relay shows up as expected. 
Am I the only one stuck at this point?

**Comment by nadav-har-tzvi** [comment]
@NFGodoy Hi and sorry for the late reply.
When you select a location in the text box at the top right corner of the UWP application, a HTTP request is sent to the  BlueYonder.Companion Web API backend, to the /Flights endpoint, with the following parameters: origin (Id), destination (Id) and the current datetime + 12 hours.
The datetime parameter wasn't timezone aware, resulting in incorrect parsing of the date on the backend.

I opened a PR that fixes this, and a few other small issues I noticed along the way. (e.g. - the notifications not actually working as expected)
